src/ex6/client/src/components/todo-element.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import { deleteItem, markItemAsOld, changeItemStatus } from "../item_client.js";
+import PropTypes from 'prop-types';
+
+export function TodoElement(props) {
+  const [todoClassName, setTodoClassName] = useState("todo-li existing-todo")
+
+  const todo = props.todo;
+  const doneTime = todo.done === null? "" : `Done at ${todo.done.slice(11,16)} ${todo.done.slice(0,10)}`;
```

Comment:
try to work with timestamps instead of date strings, use date-fns or moment for formatting

Reply:
It comes from a database as a string, despite that I create it as new Date() and save as date...